desktop/assets/texts/en-US/main_menu.ftl

```diff
@@ -0,0 +1,21 @@
+# A note on ellipsis: they indicate when an action won't immediately happen, but needs more information from the user.
+# For example, "Open File..." won't open a file, it'll ask the user to choose a file and *then* open the file.
+# Whereas "Close" or "About" wouldn't need any information, they'd immediately happen as soon as you click it.
+# In general, if clicking the thing immediately does that thing, no ellipsis. If it asks for more information, ellipsis.
+# This is the case for English, but other languages may have other rules or ways of showing this.
+
+# Some items may be also be duplicated here, like "About Ruffle" in the menu and "About Ruffle" elsewhere as the header of the about menu.
+# This is because some languages may use different wording for one or the other, especially with things like capitalisation.
```

Comment:
nit: Some context for which is which would be nice then. But this is definitely not a blocker, just a note.

Reply:
Is the file it's in and key name not enough?

`main_menu.ftl` `help-menu-about` is for the main menu item
`about_dialog.ftl` `about-ruffle` is for the about dialog
They both say "About Ruffle".

Reply:
It's probably fine, now that I think of it, translators can even request additional context on Crowdin itself. No need to get ahead of imagined problems needlessly...

Reply:
Yeah, we can also put screenshots on the items in crowdin for super duper clarity. The comment here was mostly to future developers going "why not just a single text if they're both the same", because they might not be outside of English